src/React.Tests/Mvc/HtmlHelperExtensionsTests.cs

```diff
@@ -33,7 +33,7 @@ private Mock<IReactEnvironment> ConfigureMockEnvironment()
 		public void ReactWithInitShouldReturnHtmlAndScript()
 		{
 			var component = new Mock<IReactComponent>();
-			component.Setup(x => x.RenderHtml()).Returns("HTML");
+			component.Setup(x => x.RenderHtml(false)).Returns("HTML");
```

Comment:
Can't change this, VS complains: "An expression tree may not contain a call or invocation that uses optional arguments".